docs/quick-start/connecting-mysql/page.md

```diff
@@ -1,26 +1,32 @@
-# Connecting MySQL
+# Connecting MariaDB/MySQL
 
-Just like Redis GoFr also supports connection to SQL(MySQL and Postgres) databases based on configuration variables.
+Just like Redis GoFr also supports connection to SQL(MariaDB, MySQL and Postgres) databases based on configuration variables.
 
 ## Setup
 
-Users can run MySQL and create a database locally using the following Docker command:
+Users can run MariaDB/MySQL and create a database locally using the following Docker command:
+
+```bash
+docker run --name gofr-mariadb -e MYSQL_ROOT_PASSWORD=root123 -e MYSQL_DATABASE=test_db -p 3306:3306 -d mariadb:latest
+```
+
+or for MySQL: 
 
 ```bash
 docker run --name gofr-mysql -e MYSQL_ROOT_PASSWORD=root123 -e MYSQL_DATABASE=test_db -p 3306:3306 -d mysql:8.0.30
 ```
 
-Access `test_db` database and create table customer with columns `id` and `name`
+Access `test_db` database and create table customer with columns `id` and `name`. Change mysql to mariadb as needed: 
 
 ```bash
-docker exec -it gofr-mysql mysql -uroot -proot123 test_db -e "CREATE TABLE customers (id INT AUTO_INCREMENT PRIMARY KEY, name VARCHAR(255) NOT NULL);"
+docker exec -it gofr-mariadb mariadb -uroot -proot123 test_db -e "CREATE TABLE customers (id INT AUTO_INCREMENT PRIMARY KEY, name VARCHAR(255) NOT NULL);"
```

Comment:
IMHO, `MySQL` is a more popular and used database but since we are supporting both what about we keep the docker command to use `gofr-mysql` i.e. **by default let's show the commands for MySQL db** and in a note we can mention to run `mariaDB` please change the name to this...